client/views/app/room.coffee

```diff
@@ -281,18 +281,10 @@ Template.room.helpers
 		# this is to make "bannerData" itself reactive by having it depend directly on the room data.
 		# Then, since that data gets synchronized with the server, the template will be reprocessed
 		# when the data changes.
-		accessPermissions = ChatRoom.findOne(this._id)?.accessPermissions || []
-		Template.instance().updateBannerData(accessPermissions)
-		return Template.instance().bannerData
-
-	# For helpers "classificationId" and "securityBannerText", "this" refers to what is returned
-	# from "bannerData"
-	classificationId: ->
-		return this.get 'classificationId'
-
-	securityBannerText: ->
-		return this.get 'text'
-		
+		roomData = Session.get('roomData' + this._id)
+		Template.instance().accessPermissions.set roomData?.accessPermissions
+		return Template.instance().accessPermissions
```

Comment:
I thought it would be cleaner to pass the exact context.  Also, if we ever had to, we could display more then one banner at a time if they weren't tied to the same session variable.  

Alternatively, we could do a hybrid approach where we pass the session key ('roomData'+this._id) to the security banner. then it wouldn't be tied to a specific variable, but whatever gets returned would need to have "accessPermissions" field...